app/resources/create.html.template

```diff
@@ -99,8 +94,15 @@
                   </span>
                 </div>
               {% endif %}
+
+          <div class="name section">
```

Comment:
changed all small subsection part to subsection instead of section (name subsection/ address subsection ... etc)